### ISSUE TITLE
Worked flawlessly until a few days ago.

### ISSUE THREAD

**Issue by paulandrewcrouch** [cdpuk/ha-bestway#47]
## Bestway device

Airjet

## Describe the bug

Worked flawlessly in HA for months, now doesn't show any entities.   Have re-logged in, have re-added the spa to the Bestway App (spa is online and working fine from the app).   Just not showing in HA.

Integration says it failed to initiate.

## Logs

```text

This error originated from a custom integration.

Logger: custom_components.bestway.coordinator
Source: custom_components/bestway/coordinator.py:37
Integration: Bestway (documentation, issues)
First occurred: 15:01:32 (13 occurrences)
Last logged: 15:22:22

Data update failed
Traceback (most recent call last):
  File "/config/custom_components/bestway/coordinator.py", line 37, in _async_update_data
    return await self.api.fetch_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/bestway/bestway/api.py", line 166, in fetch_data
    latest_data = await self._do_get(
                  ^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/bestway/bestway/api.py", line 418, in _do_get
    response.raise_for_status()
  File "/usr/local/lib/python3.11/site-packages/aiohttp/client_reqrep.py", line 1059, in raise_for_status
    raise ClientResponseError(
aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://euapi.gizwits.com/app/devdata/qLb2LxnphLIV6TKuu6azCf/latest')


```

**Comment by cdpuk** [comment]
Can you confirm your integration version number? This sounds a lot like the issue that was fixed in #45 and released as v1.3.1.

**Comment by paulandrewcrouch** [comment]
> Can you confirm your integration version number? This sounds a lot like the issue that was fixed in #45 and released as v1.3.1.

100% my error.   Thank you, updated and working well.

Sorry for not doing this first!!
